PipelineConfigRouter/PipelineConfigRouterApiManager.go

```diff
@@ -827,19 +817,21 @@ func HitCreateWorkflowApiWithFullPayload(appId int, authToken string) ResponseDT
 
 	createWorkflowRequestDto.AppId = appId
 	createWorkflowRequestDto.CiPipeline.Active = expectedPayload.CiPipeline.Active
-	//createWorkflowRequestDto.CiPipeline.CiMaterial[0].Source.Type = expectedPayload.CiPipeline.CiMaterial[0].Source.Type
 	fetchSuggestedCiPipelineName := HitGetPipelineSuggestedCiCd("ci", appId, authToken)
 	createWorkflowRequestDto.CiPipeline.Name = fetchSuggestedCiPipelineName.Result
 
 	fetchAppGetResponseDto := HitGetMaterial(appId, authToken)
 
-	for i, j := range fetchAppGetResponseDto.Result.Material {
+	branchValue := "main"
```

Comment:
Why we have hardcoded this value? please take this value from user via a test case and add cases for main and other branch.